projects/plugins/jetpack/extensions/blocks/subscriptions/paid-plan.js

```diff
@@ -1,36 +0,0 @@
-import { PanelBody, PanelRow } from '@wordpress/components';
-import { compose } from '@wordpress/compose';
-import { withSelect } from '@wordpress/data';
-import { __, _x } from '@wordpress/i18n';
-import GetAddPaidPlanButton from './utils';
-
-function PaidPlanPanel( { hasNewsletterPlans } ) {
```

Comment:
Should we re-add this?  I suspect we still need this for the Subscribe block?

Reply:
Do we?